elements/lisk-p2p/src/utils.ts

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { createHash } from 'crypto';
```

Comment:
can we use `hash` function from `lisk-cryptography`?

Reply:
@jondubois brought up a good point that since we only need the `sha256` hash from node's `crypto` module, we should use that instead of installing a new dependency to keep the library lightweight. The same logic would apply to your comment for `lisk-validator` below.

Reply:
I would prefer to consistently use the same hash function for all the cases.
For example if we move to other hash algorithm, we should be able to change and detect everywhere. Also, if someone is using lisk-p2p, most likely they are using lisk-cryptography and lisk-validator in some form =)